camel/models/base_model.py

```diff
@@ -138,3 +142,83 @@ def stream(self) -> bool:
             bool: Whether the model is in stream mode.
         """
         return False
+
+
+class ModelManager:
+    def __init__(
+        self,
+        models: Union[BaseModelBackend, List[BaseModelBackend]],
+        scheduling_strategy: str = "round_robin",
+    ):
+        """
```

Comment:
Please make sure the format is consistant with existing code. 
1) using r""" for comment
2) Using indents for comment content like https://github.com/camel-ai/camel/blob/master/camel/models/base_model.py#L30-L47
3) The comment should be above the __init__ method def

Reply:
Rewrote the docstring.